src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/FileFields.scala

```diff
@@ -0,0 +1,30 @@
+package uk.gov.nationalarchives.tdr.api.graphql.fields
+
+import java.util.UUID
+
+import io.circe.generic.auto._
+import sangria.macros.derive._
+import sangria.marshalling.circe._
+import sangria.schema.{Argument, Field, InputObjectType, ObjectType, fields}
+import uk.gov.nationalarchives.tdr.api.graphql.ConsignmentApiContext
+import uk.gov.nationalarchives.tdr.api.graphql.Tags.ValidateUserOwnsConsignment
+import uk.gov.nationalarchives.tdr.api.graphql.validation.UserOwnsConsignment
+
+object FileFields {
+  case class File(fileIds: Seq[Long])
```

Comment:
Are file IDs going to be UUIDs rather than longs, like in the prototype? The biggest advantage is that we'll have guaranteed unique IDs, even if we delete data from the DB (e.g. in integration/staging).

Reply:
As discussed, we're going to switch to Postgres (which was the original choice of DB) and then use UUID IDs.

Just to clarify something @MancunianSam: did you say MySQL doesn't support UUID primary keys, or is it more complicated than that? Because I've seen some docs saying that it is possible ([example](https://www.mysqltutorial.org/mysql-uuid/)).

Reply:
I didn't realise that. I thought that mysql didn't support them. Didn't you say that Postgres supports some other things which we need that mysql doesn't though?

Reply:
Here's one example: https://github.com/nationalarchives/tdr-consignment-api-data/pull/13

Reply:
Ah, fair enough, that's one we can leave out when we move to postgres.